nzshm_common/location/code_location.py

```diff
@@ -36,13 +39,95 @@ def __init__(self, lat: float, lon: float, resolution: float) -> None:
 
         self._code = f"{self.lat:.{self.display_places}f}~{self.lon:.{self.display_places}f}"
 
+    def __lt__(self, other: "CodedLocation"):
```

Comment:
Defer until we understand relationship to the current sort by `.code`

Add some tests to compare this sort to current alphabetic sort on `.code`.

Reply:
More tests added to characterize differences, and get things more consistent.